layouts/shortcodes/tab.html

```diff
@@ -0,0 +1,13 @@
+{{ if .Parent }}
+	{{ $name := trim (.Get "name") " " }}
+	{{ if not (.Parent.Scratch.Get "tabs") }}
+	{{ .Parent.Scratch.Set "tabs" slice }}
+	{{ end }}
+	{{ with .Inner }}
+	{{ $.Parent.Scratch.Add "tabs" (dict "name" $name "content" . ) }}
+	{{ else }}
+	asdf
```

Comment:
Honestly, I don't understand enough about what's even going on here to know why there is an else/end 😅 

I was mostly pruning this down to the minimal file that worked for me.